lib/Flux/LoginServer.php

```diff
@@ -150,22 +150,24 @@ public function register($username, $password, $confirmPassword, $email, $gender
 		elseif (Flux::config('UseCaptcha')) {
 			if (Flux::config('EnableReCaptcha')) {
 				require_once 'recaptcha/recaptchalib.php';
-				$resp = recaptcha_check_answer(
-					Flux::config('ReCaptchaPrivateKey'),
-					$_SERVER['REMOTE_ADDR'],
-					// Checks POST fields.
-					$_POST['recaptcha_challenge_field'],
-					$_POST['recaptcha_response_field']);
-				
-				if (!$resp->is_valid) {
+				$response;
```

Comment:
It's intended to be $response = false; basically.

I suppose I could change it to $response = $_POST["g-recaptcha-response"]; which is the client side check with the form and then change if($_POST["g-recaptcha-response"]) to if($response).